dsci522_environment.yml

```diff
@@ -0,0 +1,10 @@
+name: dsci522
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9
+  - numpy=1.22
+  - pandas=1.3
+  - matplotlib
+  - ipykernel
+prefix: /Users/shengjiayu/miniforge3/envs/dsci522
```

Comment:
Manually remove the prefix as it is specific to your local system

```suggestion
```

Reply:
please activate this environment and ensure it runs the entire notebook. I am certain **sklearn** is used to create the models but it is absent from the environment.yml

Reply:
@shengjiayuu please review the suggestions